### DISCUSSION TITLE
[Feature Request] Reply directly to comments without having to quote them

### DISCUSSION THREAD

**Discussion by theMuslimDev** [community/community#30790] [Pull Requests] [answered]
I think this will be a very useful feature. Rather then having to quote a comment and then replying to it. Replying directly to it and it being shown as a thread would be a good one. 

Example that comes to mind is when people leave review comments and you can directly reply to the particular review.

**Comment by isbadawi** [comment, marked as answer]
This is on GitHub's public roadmap: https://github.com/github/roadmap/issues/552

**Reply by rskrakowski1** [reply]
Is there a timeline of when this feature will be available? Big pull requests get out of hand quickly

**Reply by rogerbramon** [reply]
Based on the roadmap, should be during this Q4 2022.

**Reply by rskrakowski1** [reply]
Any update on this? Why has this been marked as outdated? This is is such a simple feature that is _already present_ in another area of pull requests.

<img width="683" alt="image" src="https://github.com/user-attachments/assets/394a57b9-6711-40d9-b654-29b3e5f3a0c6">
